docs/walkthrough/using-callbacks.md

```diff
@@ -126,3 +126,17 @@ The early stopping callback triggers at the end of training and evaluation batch
 - `minimum_delta`: The minimum amount of improvement that a model can have over the previous best model to be considered worthwhile, zero by default, meaning that the training will not stop early unless the performance starts to decrease
 - `patience`: The number of consecutive rounds without improvement before the training is stopped, two by default.
 - `baseline`: an optional parameter that is used to compare the model's score against instead of the best previous model when checking for improvement. This baseline does not get changed over the course of a run.
+
+
+## WiSTFTCallback

```

Comment:
WiSEFTCallback

Reply:
ah good catch, i hope i didn't make it wrong in core